content/en/tracing/profiling/java.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Profiling Java Applications
+kind: documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+The Datadog Profiling library supports the Java JRE 11 and higher of both Oracle JDK and OpenJDK. Early-access versions of Java aren't supported.
+Java profiling and tracing are shipped within the same `dd-java-agent.jar` library. In order to use tracing or profiling, you must enable these functions. To enable tracing, follow the [enable tracing documentation][1].
+
+**Note**: Some specific builds will work with Datadog profiling if they support [Java Flight Recorder][2].
+
+## Getting Started
```

Comment:
As written, the user sort of needs to mentally assemble the various steps. I think we could minimize that (small) mental burden and, more importantly, reduce onboarding friction and errors by showing the whole thing together, followed by more detail. We might also give them a heads up that it can take a couple of minutes for the profiles to show up, which has confused a couple of early adopters. Eg

> ## Getting Started
> 
> 1. Add dd-trace-java version TODO or greater to your pom.xml or equivalent. (TODO confirm details)
> 2. Update your service invocation to look like
> ```sh
> java -javaagent:dd-java-agent.jar -Ddd.profiling.enabled=true -Ddd.profiling.api-key.file=file/with/api-key -other-datadog-apm-args -jar your-service.jar -your-service-flags (TODO confirm exact syntax)
> ```
> 3. Deploy your service, give it a minute or two to upload a profile, and [check it out at in Datadog](https://app.datadoghq.com/profiling)!
> 
> Notes:
> * -javaagent needs to come first because...
> * The profiler needs an api key because...
> * You can use environment vars instead of most system properties...

Reply:
We need to fix at least some of this before we can merge.